video_worker/tests/test_video_worker.py

```diff
@@ -202,7 +191,7 @@ def change_video_invalid_intake(self):
                 # Call VideoWorker run method.
                 self.VW.run()
 
-        if not path_exists:
+        if not mock_exists.side_effect:
```

Comment:
@yro not this way, I was actually saying, change the test cases data  `@ddt`  to contain `mock_exists`'s side effect like:
```bash
(
             {
                 'encode_profile': 'static-pipeline',
                 'exists_side_effects': [True, False]
             }
)
```

then in test:
```bash
def test_run(self, mock_data, mock_mkdir, mock_exists)
    mock_exists.side_effect = mock_data.get('exists_side_effects', [True, True, True])
```